packages/web-react/src/components/Dropdown/stories/DropdownFullwidthMode.tsx

```diff
@@ -0,0 +1,29 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context';
+import { SpiritDropdownProps } from '../../../types';
+import { Stack } from '../../Stack';
+import Dropdown from '../Dropdown';
+import { dropdownContent, dropdownTrigger } from './Dropdown';
+
+const Story: ComponentStory<typeof Dropdown> = (args: SpiritDropdownProps) => (
+  <IconsProvider value={icons}>
+    <Stack>
+      Mobile only
+      <Dropdown {...args} isFullWidth fullwidthMode="mobile-only" renderTrigger={dropdownTrigger}>
```

Comment:
@adamkudrna Combination of `isFullWidth` and `fullWidthMode` makes no sense, eg. `fullWidthMode` automatically activates full width. So using `isFullWidth` with `fullWidthMode` should not be necessary.